layouts/shortcodes/note.html

```diff
@@ -0,0 +1,8 @@
+<div class="flex px-4 py-3 rounded-md bg-primary-100 dark:bg-neutral-700">
+  <span class="ltr:pr-3 rtl:pl-3 text-primary-400">
+    {{ partial "icon.html" "comment" }}
+  </span>
+  <span class="no-prose dark:text-neutral-300">
+    <strong>NOTE: </strong> {{- .Inner | markdownify -}}
```

Comment:
👋 Thank you for adding this, I was looking for such a shortcode while porting my blog to Congo.

Could this new shortcode follow the same approach as [alert](https://github.com/jpanther/congo/blob/2ceab5d4b0106549a77edcf86d4c40f301282893/layouts/shortcodes/alert.html), i.e. do not hard-code `NOTE`? 🤔

Reply:
Sure I can remove the `Note` prefix.

Just want to share on thought:
 - 👍  I like to add as many content as possible to the shortcode.
   - If you use the note/alert boxes you could end up with `ALERT: / Alert: | NOTE: / Note: ` in multiple variations
 - 👎 With the included prefix it takes the possibility for custom prefix patterns.

On either way, alert and note should be identical in this behavior.

Reply:
I believe that hard-coded shortcodes are also hard/impossible to i18n, and i18n is a first-class citizen in Congo.

Reply:
That's a good point. I have not thought about that

Reply:
So I suggest the following:
- I remove the hard coded prefix for this PR.
- I'm going to bring up another PR which includes the prefix for note, alert and other shortcodes. These will be using i18n.

What do you think?

@jpanther: Would this also work for you?

Reply:
Thanks for thinking int through! I think that your suggestion makes sense, but we'll wait for @jpanther's feedback as he's the author and maintainer.